server.go

```diff
@@ -67,13 +68,11 @@ func (s *Server) Run(port int) error {
 	s.setDefaultLimits()
 
 	router := chi.NewRouter()
-	router.Use(middleware.Throttle(s.Limits.ServerThrottle), middleware.RealIP, rest.Recoverer(s.Logger))
```

Comment:
I think we should keep throttler, realio and recoverer in the default set of middlewares. If throttle is smth not tha obvious but both realip and recoverer are must have

Reply:
i know the original ticket mentioned realip and recoverer as something user may pass, but I can't see any use case where user may want to skip these two. Well, maybe realip (not sure why, but anyway) but lack of recoverer will fail server on errors